hack/jenkins/e2e.sh

```diff
@@ -202,7 +214,19 @@ case ${JOB_NAME} in
     : ${PROJECT:="k8s-jkns-e2e-gce-flaky"}
     ;;
 
-  # Runs all non-flaky tests on GCE in parallel.
+  # Runs slow tests on GCE, sequentially.
+  kubernetes-e2e-gce-slow)
+    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-slow"}
+    : ${E2E_DOWN:="false"}
+    : ${E2E_NETWORK:="e2e-slow"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=$(join_regex_no_empty \
+          ${GCE_SLOW_TESTS[@]:+${GCE_SLOW_TESTS[@]}} \
+          )"}
+    : ${KUBE_GCE_INSTANCE_PREFIX:="e2e-slow"}
+    : ${PROJECT:="k8s-jkns-e2e-gce-slow"}
+    ;;
+
+    # Runs all non-flaky tests on GCE in parallel.
   kubernetes-e2e-gce-parallel)
```

Comment:
did we want to exclude slow tests in other builds (like this one)? or only e2e-gce for now?